### ISSUE TITLE
Сonfiguration prometheus.exporter.process

### ISSUE THREAD

**Issue by orlovds** [grafana/alloy#1935]
### What's wrong?

Good afternoon, 
please tell me how to force  prometheus.exporter.process collect data about all processes on VM?
An analog of the setting in process_exporter:
```
cat > /etc/process-exporter.yml << EOF
process_names:
 - name: "{{.Comm}}"
 cmdline:
 - '.+'
EOF
```

### Steps to reproduce

Run alloy and request data
![Image](https://github.com/user-attachments/assets/11fcc485-de9c-431a-bd87-a6dbd45ac71e)

### System information

ALT SP Server 11100-01

### Software version

alloy, version v1.4.2

### Configuration

// Process metrics
prometheus.exporter.process "local" {
  track_children = true
}

// Configure a prometheus.scrape component to collect process_exporter metrics.
prometheus.scrape "process" {
  targets    = prometheus.exporter.process.local.targets
  forward_to = [prometheus.remote_write.mimir.receiver]
}

**Comment by Nachtfalkeaw** [comment]
prometheus.exporter.process "all" {
  procfs_path = "/proc"
  track_children = true
  track_threads = true
  gather_smaps = true
  recheck_on_scrape = true

  matcher {
    name = "{{.Comm}}"
    cmdline = [".+"]
  }
}

**Comment by orlovds** [comment]
@Nachtfalkeaw  Friend, thank you so much!!! 
It's a pity that all this is not in documentation...

**Comment by Nachtfalkeaw** [comment]
Many of these components are based on standalone exporters and there is a chance to get some more information.

https://github.com/ncabatoff/process-exporter

**Comment by orlovds** [comment]
@Nachtfalkeaw Of course, but if there https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.exporter.process/#example was your example from this issues, it would be much clearer....
